lesson2/main.js

```diff
@@ -0,0 +1,17 @@
+// create list property
+const li = document.createElement("li");
+
+// create anchor property
+const a = document.createElement("a");
+a.href = "1.html";
+a.textContent = "これです";
+
+// create img property
+const img = document.createElement("img");
+img.src = "bookmark.png";
+img.alt = "ブックマーク";
+
+// add them in ul property
+const ul = document.getElementById("js-lists");
+
+ul.appendChild(li).appendChild(a).appendChild(img);
```

Comment:
If you want to place the string after the a tag, it is recommended to use prepend or insertAdjacentElement.

●Prepend
https://lab.syncer.jp/Web/API_Interface/Reference/IDL/ParentNode/prepend/

●insertAdjacentElement
https://cly7796.net/blog/javascript/insert-elements-with-javascript/

Reply:
Thank you.I thought insertAdjacentElement is easier than Prepend to rewriting this code, so I replaced `appendChild(img)` with `insertAdjacentElement("afterbegin", img)` !